posts/espp-in-this-economy/index.qmd

```diff
@@ -0,0 +1,215 @@
+
+No. **The ESPP earns about 91.57% annualized interest**.
+
+It's a much better deal.
+
+A bank account with a 15% interest rate earns the full 15% only on money present in the account for an entire year.
+
+In the ESPP, money is tied up for 3 months on average (half of the purchase period). With contributions each bimonthly paycheck, the first is escrowed for 6 months, the next for 5.5 months, etc. until the final one is only escrowed for a few days. This averages to 3 months.
+
+The result is a much higher apples-to-apples rate:
+
+$$
+\begin{flalign}
+r &= \biggl(\frac{A}{P}\biggl)^t - 1 \\
+  &= \biggl(\frac{1}{1 - discount}\biggl)^t - 1 \\
+  &= \biggl(\frac{1}{0.85}\biggl)^4 - 1\\
+  &= 0.9157 \\
+  &= 91.57\%
+\end{flalign}
+$$
+
+<small>$t=4$ because there are 4 3-month quarters in a year.</small>
+
+To calculate ESPP profit for your own plan, try the tool on the right.
+
+```{ojs}
+//| panel: sidebar
+
+html `<figcaption>ESPP Interest Rate Calculator</figcaption>`;
+
+viewof purchase_period_in_months = Inputs.select([3, 6, 12], {
+  label: "Purchase Period",
+  format: (m) => `${m} months`,
+  value: 6,
+});
+
+viewof discount_percent = Inputs.number([0, Infinity], {
+  label: "Discount (%)",
+  step: 1,
+  value: 15,
+});
+
+value_of_stock_per_dollar_contributed = (1.0 / (1.0 - (discount_percent / 100.0)));
+
+html `<hr>`;
+html `For every <strong>$1</strong> contributed, <strong>$${value_of_stock_per_dollar_contributed.toFixed(2)}</strong> worth of company stock is purchased`;
+
+average_escrow_time_in_months = purchase_period_in_months / 2.0;
+average_escrow_time_in_years = average_escrow_time_in_months / 12.0;
+interest_rate = Math.pow(value_of_stock_per_dollar_contributed, 1.0 / average_escrow_time_in_years) - 1.0;
+
+html `<hr>`;
+html `On average, money is escrowed for <strong>${average_escrow_time_in_months} months</strong>`;
+html `<hr>`;
+html `Annualized interest rate: <strong class="fs-1">${(100.0 * interest_rate).toFixed(2)}%</strong>`;
+```
+
+## Lookback
+
+The best ESPPs offer a lookback feature.
+
+With a lookback, the discount is applied to the price at the start of an offering period or the price on the purchase date, _whichever is lower_.
+
+"Whichever is lower" makes it difficult to lose.
+
+When the stock rises in value, shares are purchased at a discount from the earlier price, leading to even higher built-in profit.
+
+When the stock falls in value, shares are purchased at a discount from the current market price on the purchase date.
+
+There is profit in either scenario as long as the shares are sold immediately to lock it in.
+
+To calculate ESPP profit for your own plan that has a lookback feature, try the tool on the right.
+
+Try different scenarios, including ones where the stock loses value over the purchase period.
+
+```{ojs}
+//| panel: sidebar
+
+html `<figcaption>ESPP Interest Rate Calculator</figcaption>`;
```

Comment:
Could you name this one "ESPP Interest Rate Calculator with Lookback" to differentiate it from the previous tool?